charts/csm-application-mobility/values.yaml

```diff
@@ -8,39 +8,51 @@ image:
   pullPolicy: IfNotPresent
 
 controller:
-  image: dellemc/csm-application-mobility-controller:v0.3.0
+  image: dellemc/csm-application-mobility-controller:latest
 
 # csm-application-mobility requires cert-manager. If cert-manager is not already present in cluster, set enabled to true to install it too.
 cert-manager:
-  enabled: false
```

Comment:
Can we keep the default behaviour as false? Docs currently show default as false (https://dell.github.io/csm-docs/docs/applicationmobility/deployment/#configuration)